cli/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awareno-pattern.yaml-dependency_aware/results.txt

```diff
@@ -21,6 +21,9 @@ SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERS
 === stderr - plain
 Nothing to scan.
 
+Some files were skipped or only partially analyzed.
+  Scan skipped: 1 files matching .semgrepignore patterns
```

Comment:
Is this output expected?

Reply:
yes. That test runs on the whole set of targets, and it ignores `node_modules` (which it should). The new output is because lockfiles are getting ignored properly now. Before it was an ad-hoc check that didn't update this info when printing results.